cmake/dependencies/PICSAR.cmake

```diff
@@ -11,18 +11,32 @@ function(find_picsar)
         set(CMAKE_POLICY_DEFAULT_CMP0077 NEW)
 
         # Enable or disable QED lookup tables generation
-
-        # If table generation is enabled, enable or disable
-        # openMP support depending on WarpX_COMPUTE
-        if(WarpX_QED_TABLE_GEN)
+        if(WarpX_QED_TABLE_GEN OR WarpX_QED_TOOLS)
             set(PXRMP_QED_TABLEGEN ON CACHE INTERNAL "")
-            if(WarpX_COMPUTE STREQUAL OMP)
-                set(PXRMP_QED_OMP ON CACHE INTERNAL "")
+        else()
+            set(PXRMP_QED_TABLEGEN OFF CACHE INTERNAL "")
+        endif()
+
+        # Add QED tools folder
+        if(WarpX_QED_TOOLS)
+            add_subdirectory(Tools/QedTablesUtils)
+        endif()
```

Comment:
This needs to go further down, otherwise the tool will not be build if PICSAR is not a superbuild (but an external, pre-installed dependency).